gradio/routes.py

```diff
@@ -308,15 +320,13 @@ def get_types(cls_set: List[Type], component: str):
 
 def get_state():
     raise DeprecationWarning(
-        "This function is deprecated. To create stateful demos, pass 'state'"
-        "as both an input and output component. Please see the getting started"
-        "guide for more information."
+        "This function is deprecated. To create stateful demos, use the Variable"
+        " component. Please see the getting started for more information."
     )
 
 
 def set_state(*args):
     raise DeprecationWarning(
-        "This function is deprecated. To create stateful demos, pass 'state'"
-        "as both an input and output component. Please see the getting started"
-        "guide for more information."
+        "This function is deprecated. To create stateful demos, use the Variable"
```

Comment:
I think we should leave the old warnings. If someone is using `gr.set_state()` and `gr.get_state()`, they are probably trying to create an `Interface`, not a general `Blocks`